crate_universe/src/splicing.rs

```diff
@@ -582,4 +584,35 @@ mod test {
             PathBuf::from("/tmp/abs/path/workspace/.cargo/config.toml"),
         )
     }
+
+    #[test]
+    fn splicing_metadata_workspace_path() {
```

Comment:
Can you explain how this tests the issue you're seeing? At a glance it seems unrelated.

Reply:
Awesome! Thank you!